### DISCUSSION TITLE
Atlas of Hand Bone Age( 성장기 범주별 특징 )

### DISCUSSION THREAD

**Discussion by SHEYCHAE** [kikiru328/Bone_Detection#39] [Domain]
[Atlas_of_Hand_Bone_Age.pdf](https://github.com/kikiru328/Bone_Detection/files/7395522/Atlas_of_Hand_Bone_Age.pdf)

## 뼈 발달 

<img src="https://user-images.githubusercontent.com/77530039/138297639-1a11960e-6119-4238-8795-545125d5e4ee.png" width="600" height="300"/>

## 성장기의 골격 성숙도 지표
The purpose of this section is to describewhich bones in the hand and wrist
are the most suitable indicators of skeletal maturity during the different
phases of postnatal development. In the majority of healthy children, there
is an established sequence of ossification for the carpal (Figure 3), metacarpal
and phalangeal bones, which is remarkably constant and the same for
both sexes. Overall, the first ossification center to appear in hand and wrist
radiographs is the capitate, and the last is, most often, the sesamoid of the
adductor pollicis of the thumb

손과 손목의 어떤 뼈가 산후 발달의 다른 단계 동안 골격 성숙의 가장 적합한 지표인지를 설명
대부분의 건강한 아동에서 손목뼈, 중족골, 지골뼈의 골화 염기서열이 확립되어 있으며, 이는 남녀 모두에게 매우 일정하고 동일
전체적으로, 손과 손목 방사선 사진에 나타나는 첫 번째 골화 센터는 카피테이트이고, 마지막은 엄지손가락의 보조자 폴리시스(Adductor Policis)의 세사모이드이다.

The first epiphyseal center to appear is that of the distal radius, followed by those of the proximal phalanges, the metacarpals, the middle phalanges, the distal phalanges, and, finally, the ulna. There are, however, two main exceptions to this sequence: the epiphysis of the distal phalanx of the thumb commonly appears at the same time as the epiphyses of the metacarpals, and the epiphysis of the middle phalanx of the fifth finger is frequently the last to ossify. 
Since the predictive value of the ossification centers differs and changes during growth, the reviewer should primarily focus on the centers that best characterize skeletal development for the 
subject’s chronological age.

가장 먼저 나타나는 착시골 중심은 말단 반지름이고, 그 다음으로 근위 지골, 중족골, 중간 지골, 말단 지골, 그리고 마지막으로 척골의 중심이다. 
그러나 이 순서에는 두 가지 주요 예외가 있다. 엄지손가락의 말단 팔랑뼈의 에피프 투석은 중족골의 에피피스와 동시에 나타나며, 다섯 번째 손가락의 중간 팔랑뼈의 에피프 투석은 종종 마지막으로 골화된다.
골화 센터(중심?)의 예측값을 다르고 성장 중에 변화하기 때문에, 우선 대상의 연령에 맞는 골격발달을 가장 잘 특징짓는 중심부에 초점을 맞춰야 한다.
<img src="https://user-images.githubusercontent.com/77530039/138298912-5b03a4e5-f8f1-4a4b-8546-67dbdae0592e.png" width="300" height="300"/>

capitate (1) 알머리뼈 유두골
hamate(2) 갈고리뼈 유구골
triquetral (3) 세모뼈
lunate (4) 반달뼈, 월상골
trapezium (5) 큰마름골, 대능형골
trapezoid (6) 작은마름골, 소능형골
navicular or scaphoid (7) 발배뼈(손배뼈?), 주상골
pisiform (8) 콩알뼈, 두상골

-----------------------------------------------
### 골연령평가 

1) Infancy (the carpal bones and radial epiphyses);
2) Toddlers (the number of epiphyses visible in the long bones of the
hand);
3) Pre-puberty (the size of the phalangeal epiphyses);
4) Early and Mid-puberty (the size of the phalangeal epiphyses);
5) Late Puberty (the degree of epiphyseal fusion); and,
6) Post-puberty (the degree of epiphyseal fusion of the radius and ulna).

골연령 평가를 용이하도록 하기 위해 골격 발달을 6가지 주요 범주로 나누고
각 그룹의 골격 성숙도를 가장 잘 예측하는 특정 골화는 괄호로 넣음 
1. 영아기( 손목뼈와 방사상 골단) 
2. 유아 ( 손의 긴뼈..? 에서 볼 수 있는 골단의 수)
3. 사춘기 전 ( 지골 골단의 크기)
4. 사춘기 초기 ( 지골 골단의 크기)
5. 후기사춘기 ( 골단 융합의 정도 )
6. 사춘기이후 (요골과 척골의 골단 융합 정도)

-----------------------------------------------
### 1. 영아기(여: 출생~10개월 / 남: 출생~14개월)
- 만삭 신생아의 모든 수근골,지골,중수골,요골 및 척골의 모든 골단은 골화가 없다. 
- capitate와 hamate의 골화 중심은 생후 약 3개월에 명백해지고, 다음 6개월동안 유일하게 관찰 가능한 유용한 특징으로 남음
- 여아는 약 10개월, 남아는 약1세 3개월에 요골의 말단 골단에 작은 골화 센터가 나타남
- 골화 센터가 없어서 영아기에 손과 손목 방사선 사진을 이용한 골격 성숙도 평가는 어려움 
- 생후 첫 해의 뼈 성숙도를 추정하려면 상지와 하지에 있는 이차 골화센터의 수, 크기 및 구성을 평가해야 함

<img src="https://user-images.githubusercontent.com/77530039/138296301-eb84bdea-9d69-4de5-8233-ae26266b2d1e.png" width="300" height="400"/>

- 유아기에 골연령은 주로 capitate, hamate, 반지름의 말단 epiphysis유무에 기초
- capitate는 보통 hamate보다 조금 일찍 나타나며, 골화 중심과 둥근 모양을 가지고 있다. 말단 방사상 epiphysis은 나중에 나타남

### 2. 유아기 (여: 10개월 ~2살 / 남: 14개월~3살)
- 모든 지골과 중수골 골단의 골화 센터는 유아기에서 인식할 수 있게 되며, 일반적으로 가운데 손가락이 먼저이고 다섯번째 손가락이 마지막임
- 골연령 결정은 주로 규칙적인 특정 패턴으로 나타나며, (식별 가능한) 골단 골화 센터의 수에 대한 평가를 기반으로 함 
   * 1. 근위 지골의 골단 - 2. 중수골의 골단 - 3. 중간지골의 골단 - 4. 말단 지골의 골단
   * ( 일반적으로 남아의 경우 1년 3개월, 여아의 경우 1년 6개월에 인지되는 엄지 손가락 원위 지골의 골화 중심의 조기 출현은 예외 )
   * ( 마지막 지골 골단이 나타나는 다섯번재 손가락 중지골의 골화 센터..중심?이 늦게 나타나는 것도 예외 ) 
- 유아기에서는 손목에 있는 수근골의 성숙정도와 수가 덜 유용함 .왜냐하면 3개 또는 4개(capitate, hamate
and lunate and, at times, trapezoid)만 인식할 수 있기 때문 

<img src="https://user-images.githubusercontent.com/77530039/138300233-86ef75fb-a921-4a06-841e-168776bcb189.png" width="300" height="400"/>
- 유아기에서 골연령은 주로 지골과 중수골에서 인식할 수 있는 골단 골화 센터의 수를 기반으로 함

### 3. 사춘기 전 (여: 2~7살 / 남:3~9살)
- 사춘기 이전 아동의 골격 성숙도 평가는 인접 골간단과 관련된 지골의 골단 크기를 기본으로 함
- 골단의 골화중심은 너비와 두께가 증가하고 결국 골단부만큼 너비가 넓은 가로 지름을 가정함 
- 중가 지골보다 원위 지골의 골단 크기에 더 많은 가중치가 부여되고 근위지골의 골단 크기에는 훨씬 더 가중치가 부여됨 
- 하지만 말단 지골의 발달은 여러 연령대에서 유사하게 나타나기 때문에 중간 지골의 후두엽의 성숙 정도에 따라 평가하기도 함. (드문 경우, 근위뼈의 발달이 평가에 포함될 수 있음)
- 척골과 모든 수근골의 골단은 pisiform을 제외하고 일반적으로 사춘기 이전에 알아볼 수 있다. but 중수골과 같은 골화센터는 이 단계의 뼈 나이를 나타내는 신뢰할 수 있는 지표가 아님 

<img src="https://user-images.githubusercontent.com/77530039/138301728-e878230a-916e-4215-97bd-6ab4b109f718.png" width="400" height="300"/>
- 이 발단 단계에서 형이상학만큼 넓어지는 골단 폭의 점진적 성장 묘사

<img src="https://user-images.githubusercontent.com/77530039/138301795-4ca55329-4558-495c-a623-2b7894d9534f.png" width="300" height="500"/>
- 골연령 평가는 주로 원위 및 중간 지골에서 더 작은 골단과 더 큰 골단 사이의 너비 차이 정도를 기반으로 함 

### 4. 사춘기 초반과 중반 (여:7~13/ 남: 9~14)
- 사춘기 이전 어린이와 마찬가지로 사춘기 초기 및 중반의 골격 성숙도 평가는 말단 지골(첫번쨰)와 중간지골(두번째)에 있는 골단의 크기를 기반으로 함. 이 단계의 골단은 계속해서 성장하고 그 폭은 매우 커짐
- 그 후, epiphyses의 윤곽은 metaphyses와 겹치거나 덮기 시작합니다. 
- 이 캡핑 효과는 2차원 방사선 촬영에서 작은 뿔과 같은 작은 뼈들이 축의 양쪽에 자라나는 것으로 묘사됨
- 첫 번째 중수골의 머리 바로 중앙에 있는 abductor pollicis의 힘줄에 있는 pisiform과 sesamoid는 사춘기 동안 알아볼 수 있게 됨
- 그러나, 다른 carpals나 중족골(metacarpals)과 마찬가지로 이 중심들은 이 발달 단계에서 뼈의 나이를 나타내는 지표로서 신뢰성이 떨어진다.

<img src="https://user-images.githubusercontent.com/77530039/138303383-dd26eec4-ea5a-4bac-902a-e0a2030e5cbd.png" width="500" height="300"/>
- 골단의 점진적 성장 묘사
- 골단 융합 이전에 골단과 겹치며 골단의 양쪽 끝에 작은 뿔 모양의 구조를 묘사하는 골단 모양에도 특별히 주의를 기울일 것 (맨오른쪽)

<img src="https://user-images.githubusercontent.com/77530039/138303644-0de4dd6d-13a6-4fa9-91c6-8bae7c075bae.png" width="300" height="500"/>
- 평가 주로 말초 및 중간지골 기반으로 

### 5. 늦은 사춘기 (여:13~15 / 남: 14~16)
- 골격 성숙도 평가 주로 말단 지골의 골단 융합정도를 기반으로함 
- 손의 긴 뼈?에서 골단과 골단의 융합은 규칙적인 특정 패턴으로 발생하는 경향이 있음
   * 1. 말단 지골의 융합 - 2. 중수골의 융합 - 3. 근위 지골의 융합 - 4. 중간 지골의 융합 
- 쟤네 형태학 때문에, 후두엽(metacarpals)의 후두엽 융접(epiphyseal fusion)은 방사선 사진으로는 잘 묘사되지 않으며, 따라서 지골에서의 융접 정도에 더 큰 관심이 쏠림
- 손목뼈 필요없음

<img src= "https://user-images.githubusercontent.com/77530039/138308009-8cb86cc2-8a7d-4afc-8318-4565bc590869.png" width="400" height="300"/>
- 일반적으로 physis의 중심에서 시작되는 epiphyses와 metaphyses의 점진적 융합 정도(왼->오)

<img src= "https://user-images.githubusercontent.com/77530039/138308029-56da4a88-cf66-4a72-9a2c-b73d181f2a11.png" width="400" height="300"/>
- 성숙도 평가 지골의 골단 융합 정도(첫 번째)와 중간지골의 융합 정도(두 번째)를 기반으로

### 6. 사춘기 이후 (여: 15~17/ 남:17~19)
- 사춘기 이후에는 모든 수근, 중수골 및 지골이 완전히 발달하고 신체가 닫힘
- 골격 성숙도 평가는 척골과 요골의 골단 융합 정도를 기반으로 함

![image](https://user-images.githubusercontent.com/77530039/138309416-d912b050-56da-4e66-871a-260f368949f2.png)
- 척골과 요골 골단의 점진적 융합 정도 (왼->오)

<img src= "https://user-images.githubusercontent.com/77530039/138310985-60f58a09-0288-4d7d-a7c9-84ee51e71ddb.png" width="300" height="500"/>
- 이 발달 단계에서 골격 성숙은 가장 먼저 발생하는 척골의 골단 융합과 요골을 기반으로 함

**Comment by MonicaKim89** [comment]
bone ossification의 정의 잘 봤습니다~
bone ossification과 그걸 통해 골연령을 분석하는 방법이 여러가지가 있어요
대부분 TW3를 사용하고
한국소아에 대한 골연령관련 논문이 있네영
https://pc.kjronline.org/DOIx.php?id=10.3348/kjr.2015.16.1.201

한번봐보세여!

**Reply by SHEYCHAE** [reply]
감사합니다!!!!!!!

**Comment by SHEYCHAE** [comment]
# 왜 TW3를 사용할까 ? 

## GP방법
- GP 방법은 왼쪽 손목사진을 (연령별로 표본 방사선 사진이 나열되어있는) GP도감과 대조하여 골연령 측정 
- GP 도감 기준은 1930년대 미국의 소수 중상위 계층 소아들로부터 만들어짐
빠르고 간편함

## TW방법

- 손목과 손의 뼈들을 TW도감에 나와있는 각 부위별 뼈 성숙도에 따른 등급을 매기고 이에 따른 각각의 점수를 합산하여 골연령을 측정
- 1950년대 영국의 아동들을 대상으로 개발되었고, 소아들의 성장과 골성숙이 빨라지는 추세를 반영하여 TW2-TW3로 개정됨 
( TW3방법은 1980-1990년대 ( 다른논문에서 2001년이라고함)북미,유럽 소아들을 대상으로 했고 성숙도 점수 합산 방법도 간략해짐 ) (TW3는 RUS점수만으로 계산함) 
- 시간이 좀 걸림 

## 실험 결과
- 골성숙은 키와 성장속도처럼 유전적,환경적 영향의 상호작용을 반영하며 인종에 따라 차이를 보인다.
- 각 인구집단들은 주어진 나이의 평균 뼈 성숙도와 연령이 증가하면서 성숙하는 패턴이 다를 수 있기 대문에 골연령에 대한 표준치는 관련된 각 개별 인구집단에 대해 각각 확립되어야함.
- 또한 소아들은 이전 세대보다 더 빨리 성숙하므로 골성숙의 표준치가 시대에 따라 개정되어야함.
- 따라서 GP방법은 현재 아동의 골연령과 차이가 있을 수 있dma 
- TW3 방법은 GP 방법돠 골연령이 같거나 오히려 조금 더 어리게 측정되는 경향을 보임 ( 이전 세대보다 더 빨리 성숙하므로 gP보다 어리게 측정되는게 더 좋은 것 같음 )
- 실제로 중국 소아 대상으로 한 연구에서 골성숙이 (여아의 경우 10세까지/ 남아의경우 12세까지) TW3 표준치와 일치하며 일본의 표준치와도 비슷하다고 함 
- 따라서 TW3가 TW2에 비해 동연인 골연령 측정에 더 적합하다고 여겨짐

**Comment by SHEYCHAE** [comment]
#### GP방법으로 측정한 연령 - 골연령 상관관계 (total/ boys/ girls)
![image](https://user-images.githubusercontent.com/77530039/138547211-e06f30bb-6021-46c6-8921-292afc680fce.png)

#### TW3방법으로 측정한 연령 - 골연령 상관관계 (total/ boys/ girls)
![image](https://user-images.githubusercontent.com/77530039/138547213-f84939c5-32c9-4258-b248-fa67235d5fdc.png)

#### KS방법으로 측정한 연령 - 골연령 상관관계 (total/ boys/ girls)
![image](https://user-images.githubusercontent.com/77530039/138547216-20e499ce-cfbb-4d43-a6ec-8e9c2eeebfc7.png)

## 결과
- 전체군과 여아군에서 KS방법이 다른 방법보다 상관계수가 높게나옴 ( 결정계수 0.89 ) (GP,TW3 결정계수 총0.88 , 여아0.87 ) 
- 남아군에서도 KS 방법이 TW3보다 높게나옴 (결정계수 0.9)(TW는 결정계수0.88)
- 하지만............일원분산분석에 따르면 세 가지 방법의 차이는 전체 그룹에서 통계적으로 유의하지 않음(pvalue= 0.9), 성별별로도 (여아 p=0.34/ 남아 p=0.29)

- 관찰자의 뼈 연령 평가 평균 차이는 GP 방식이 0.45±1.79개월, 
- TW3 방식이 0.45±1.81개월, 
- KS 방식이 0.21±1.19개월....  ( KS 방식 좋은데?? ) 
- 관찰자간 신뢰도 계수는 세 가지 방법 모두에서 0.95 이상이었다 (0.996에서 1.000까지)

---------------------------------------------------------------
## Duscussion
- TW3 방식은 뼈의 상세한 형상분석을 사용해 골화 부위별로 점수를 매기고 최종 합계를 플로팅하여 TW3 방식에 의한 골연령 평가를 수행함. 따라서 연구자들은 TW3방법이 높은 재현성을 제공하고 관찰자간 차이(10-12)를 줄인다고 주장하지만 시간이 많이 걸리고 적용이 어려움

- 일부 방사선 전문의와 임상의는 GP와 TW3가 현재 한국 아이들의 골격 성숙도 평가에 적용가능한지에 대해 의구심을 갖음. (한국 아이들을 기준으로 측정한 것이 아니기 때문) ( 인종에 따라 유의미한 차이를 보이니까 문제가 될 수 있음 )

#### About KS ####
-  건강한 한국 어린이 3407명의 왼손 방사선 사진에서 추출한 KS를 설립
- gp 방법과 마찬가지로 KS 골연령 추정은 아이의 왼쪽손목 방사선 촬영과 KS차트를 비교해서 실시하므로 외래 환자 환경에서 적용이 쉽고 편리함
- KS는 GP나 TW3에 비해 연륜에 따른 상관관계가 높은 것으로 나타났지만 그 차이는 크지 않았다.

- 현재 연구에는 몇 가지 제한이있음 .. 먼저 대상자가 적었고 청소년과 취학 전 아동 등 연령대가 연구에 포함되지 않았고. 또한, 뼈 연령 추정에 대한 연구에서 관찰자 내 변동성이 중요하지만, 이 연구는 관찰자 내 변동성을 포함하지 않았음
- 결론적으로, KS, GP, TW3 방법은 한국 아이들의 뼈 나이를 평가하는 데 유용하다.
 KS가 우리나라 아동의 성숙상태에서 방사선 지수로 활용되기 위해서는 인구가 많은 다기관 연구가 필요할 수 있다. 
(TW3 RUS 방식에 기초한 컴퓨터 보조 골격 나이 점수 등 신뢰할 수 있는 컴퓨터 보조 골격 나이 추정을 확립하려는 세계적인 노력이 있는 만큼 ) 한국 아이들의 컴퓨터 보조 골격 나이 추정을 지표로 삼기 위한 KS 강화와 수정을 고려해야 한다.

**Reply by SHEYCHAE** [reply]
이거 지금 당장 필요한건 아닌거같고 우리는 code breaking이 먼저지만 나중에 보기 편하라고 정리해서 올릴게 
이거는 MonicaKim님이 코맨트 주신 논문 간단히 정리해 본거고 
당장 보려고 올린거아님!!!!! 코드브레이킹 먼저 빨리 해서 이미지 전처리 먼저 열시미해보자 
GP TW KS 내용 궁금할때 들어와서 보면 됨

**Comment by kikiru328** [comment]
![image](https://user-images.githubusercontent.com/60537388/138818766-cd6c187c-0b1b-4085-950b-8a9694bee5d1.png)
